packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -124,6 +125,10 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       ...(!rtl && {
         transform: `rotate(${rotate}deg)`,
       }),
+
+      [`&.${Avatar.slotClassNames.image}`]: {
+        ...(isFontBased && getAvatarFontIconStyles(size, v)),
+      },
```

Comment:
Actually, this is a single thing that stops me from the approve button.

I understand _why_ `Avatar` can know about `Icon`, because it's a slot of `Avatar`. But, I want to avoid the reserve coupling because `Icon` is a primitive component  🤔